nomad/grapl-core.nomad

```diff
@@ -980,22 +1022,24 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image = "${var.container_registry}grapl/sysmon-generator:${var.sysmon_generator_tag}"
+        image = "${var.container_registry}grapl/${var.container_repo}sysmon-generator:${var.sysmon_generator_tag}"
+      }
+
+      template {
+        data        = local.conditionally_defined_env_vars
+        destination = "sysmon.env"
+        env         = true
       }
 
       env {
-        DEST_BUCKET_NAME            = var.unid_subgraphs_generated_bucket
-        DEAD_LETTER_QUEUE_URL       = var.sysmon_generator_dead_letter_queue
-        SOURCE_QUEUE_URL            = var.sysmon_generator_queue
-        GRAPL_AWS_ENDPOINT          = local.aws_endpoint
-        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
-        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
-        AWS_DEFAULT_REGION          = var.aws_region # boto3 prefers this one
-        AWS_REGION                  = var.aws_region
-        REDIS_ENDPOINT              = local.redis_endpoint
-        GRAPL_LOG_LEVEL             = var.rust_log
-        RUST_LOG                    = var.rust_log
-        RUST_BACKTRACE              = local.rust_backtrace
+        DEST_BUCKET_NAME      = var.unid_subgraphs_generated_bucket
+        DEAD_LETTER_QUEUE_URL = var.sysmon_generator_dead_letter_queue
+        SOURCE_QUEUE_URL      = var.sysmon_generator_queue
+        AWS_DEFAULT_REGION    = var.aws_region
+        REDIS_ENDPOINT        = local.redis_endpoint
+        GRAPL_LOG_LEVEL       = var.rust_log
```

Comment:
I think the `GRAPL_LOG_LEVEL` variable only applies to Python services; the Rust ones currently use `RUST_LOG`.

Long term, we should probably work to harmonize these as much as possible. In the meantime, we should not add unnecessary variables, since that just increases the cognitive overhead.